sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_file_system_client.py

```diff
@@ -265,7 +265,7 @@ def create_file_system(self, metadata=None,  # type: Optional[Dict[str, str]]
                                                        public_access=public_access,
                                                        **kwargs)
 
-    def create_if_not_exists(self, **kwargs):
+    def create_file_system_if_not_exists(self, **kwargs):
         # type: (Any) -> None
         """Creates a new file system under the specified account.
```

Comment:
It won't let me create a comment in code not included in the default view but below here for `public_access`, please replace `type` with `paramtype`

Reply:
Actually, you can just move the type into the declaration since there is no `or` so just `:keyword ~azure.storage.filedatalake.PublicAccess public_access:` on line 277 and remove line 279.